storagemanager/users/admin.py

```diff
@@ -1,5 +1,71 @@
+from django import forms
 from django.contrib import admin
-from .models import Profile
+from django.contrib.auth.models import Group
+from django.contrib.auth.admin import UserAdmin as BaseUserAdmin
+from django.contrib.auth.forms import ReadOnlyPasswordHashField
+from django.core.exceptions import ValidationError
+from .models import User
 
 
-admin.site.register(Profile)
+class UserCreationForm(forms.ModelForm):
+    password1 = forms.CharField(label='Password', widget=forms.PasswordInput)
+    password2 = forms.CharField(
+        label='Password confirmation', widget=forms.PasswordInput)
+
+    class Meta:
+        model = User
+        fields = ('email',)
+
+    def clean_password2(self):
+        password1 = self.cleaned_data.get("password1")
+        password2 = self.cleaned_data.get("password2")
+        if password1 and password2 and password1 != password2:
+            raise ValidationError("Passwords don't match")
+        return password2
+
+    def save(self, commit=True):
+        user = super().save(commit=False)
+        user.set_password(self.cleaned_data['password1'])
+        if commit:
```

Comment:
I think commit will always be true here so you don't need this condition? Unless you want to explicitly call this method somewhere.

Reply:
This is taken from Django Docs, and after removing `commit = True` from that line, there is an exception while adding new user in admin panel.

Reply:
I didn't mean removing `commit = True`, I'm refering to last line so `if commit:` :)